apps/greencheck/swagger.py

```diff
@@ -18,7 +18,7 @@
 
 schema_view = get_schema_view(
     openapi.Info(
-        title="Welcome to The Green Web Foundation Partner API ",
+        title="Welcome to Green Web Foundation Partner API ",
```

Comment:
Same here as above. Can we please keep the lower case 'the' here, like we do in other places where we refer to the Green Web Foundation in the middle of a sentence?